app/src/backend/GlobalAPIService.ts

```diff
@@ -13,6 +13,9 @@ export const getGrowthRatesFromOC = async (
     const response = await fetch(URL);
     logger.info(`${URL} Response Status: ${response.status}`);
     const data = await response.json();
+    if (response.status !== 200) {
```

Comment:
Would be good to also rename the method name or remove the "fromOC" suffix from it (since we are apparently loading the data from the global API and not OpenClimate).

Reply:
Will handle this on a separate branch